### PR TITLE
OBJ-237 Create new no stike off screen and controller

### PR DESCRIPTION
**Pull request by mwestacott**

### CODE REVIEW

File: src/controllers/notice.expired.controller.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {
   retrieveCompanyProfileFromObjectionSession,
 } from "../services/objection.session.service";
 
-export const get = async (req: Request, res: Response, next: NextFunction) => {
+export const get = (req: Request, res: Response, next: NextFunction) => {
```
Comment (mwestacott) [inline src/controllers/notice.expired.controller.ts:8]: Lint was showing up the presence of 'async' as not needed/a violation. Removed it and all seems to still work.

File: views/notice-expired.html [hunk 2]
```diff
@@ -20,7 +20,7 @@ <h1 class="govuk-heading-xl">
           to find out more about restoring a company name to the register.</p>
 
       <p>If this is the wrong company, start the
-        <a href="/company-number">Apply to object to a company being struck off</a>
+        <a href="/strike-off-objections">Apply to object to a company being struck off</a>
```
Comment (mwestacott) [inline views/notice-expired.html:23]: This was a broken link and wasn't going back to the start of the journey, as required in story. Fixed it here and on the new screen.

**Review by ICoxCH — APPROVED** [review]

File: src/controllers/no.strike.off.controller.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { Session } from "ch-node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+import {
+  retrieveCompanyProfileFromObjectionSession,
+} from "../services/objection.session.service";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
```
Comment (markpit) [inline src/controllers/no.strike.off.controller.ts:8]: you can probably remove NextFunction as well and its import

File: src/controllers/notice.expired.controller.ts (hunk 1 above)
Comment (markpit) [reply src/controllers/notice.expired.controller.ts:8]: you can probably remove NextFunction as well and its import

File: views/no-strike-off.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        There is no strike off notice for the company
+      </h1>
+      <p class="govuk-body-l">Check the status of {{ companyName }}
+        again in approximately 4 weeks' time. If dissolution action has started, you can use this service to apply
```
Comment (markpit) [inline views/no-strike-off.html:14]: weeks' - the apostrophe looks wrong, looks like weekses.

Comment (mwestacott) [reply views/no-strike-off.html:14]: I agree, but it's in the screenshot. Leave for now I say but will raise this with Sarah and Karen.

**Review by markpit — APPROVED** [review]

File: views/no-strike-off.html (hunk 4 above)
Comment (mwestacott) [reply views/no-strike-off.html:14]: Checked with Sarah and also consulted Google. It's needed - https://www.sarahtownsendeditorial.co.uk/2015/11/using-an-apostrophe-time-expressions/.

**Pull request merged**